code/game/turfs/turf.dm

```diff
@@ -169,6 +171,8 @@
 		air_master.remove_from_active(src)
 
 	var/turf/W = new path(src)
+	if(!defer_change)
+		W.AfterChange()
 
 	if(istype(W, /turf/simulated))
 		W:Assimilate_Air()
```

Comment:
I know, I should have probably reported that to you. Reverting the
map-loader is kinda the best solution, but then the ruins just don't load.
I don't really know what should be done.